tests/functional/fileupload_remote_test.php

```diff
@@ -100,8 +100,8 @@ public function test_successful_upload()
 		$upload = new \phpbb\files\upload($this->filesystem, $this->factory, $this->language, $this->php_ini, $this->request, $this->phpbb_root_path);
 		$upload->set_error_prefix('')
 			->set_allowed_extensions(array('gif'))
-			->set_max_filesize(1000);
-		$file = $upload->handle_upload('files.types.remote', self::$root_url . 'styles/prosilver/theme/images/forum_read.gif');
```

Comment:
@marc1706 because for some reason we are testing upload of the forum_read.gif to the files folder which does not make sense as this will no longer be a a file in 3.3 anyway so I just uploaded a random gif for performing these kind of tests to develop